lms/djangoapps/support/static/support/jsx/entitlements/components/Main/Main.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 
 import { StatusAlert } from '@edx/paragon';
 import SearchContainer from '../Search/SearchContainer.jsx';
+import EntitlementSupportTableContainer from '../Table/EntitlementSupportTableContainer.jsx';
```

Comment:
Probably a little late in the game to ask this but does platform not support JSX file imports without the `.jsx` extension?

Reply:
It works for the build process but theres a quality complaint around it.

Reply:
We could talk about removing it from the linter if its not what we like? not sure who I would bring that up to, (looping in @arizzitano for input)